packages/block-editor/src/components/inner-blocks/index.js

```diff
@@ -161,6 +162,7 @@ InnerBlocks = compose( [
 			blockListSettings: getBlockListSettings( clientId ),
 			hasOverlay: block.name !== 'core/template' && ! isBlockSelected( clientId ) && ! hasSelectedInnerBlock( clientId, true ),
 			parentLock: getTemplateLock( rootClientId ),
+			enableClickThrough: isNavigationMode() || isSmallScreen,
```

Comment:
Now the question is how we should approach mobile in this context. Should we still offer the option to switch between tools or should we hide them and provide the unique configuration tailored to touch screens.

Reply:
I have on my ToDo list to thoroughly review this, because I love this PR. But this just a quick note:

> Now the question is how we should approach mobile in this context. Should we still offer the option to switch between tools or should we hide them and provide the unique configuration tailored to touch screens.

It could be worth exploring this at some point. But there are some other aspects that probably should precede this. Right now the mobile interface is barely usable, and we need to "free up some space" to enable this to happen. ios 13 may enable a top toolbar, which could help, and breadcrumbs have not yet been brought to that interface — I know @iamthomasbishop has thought about how to do that. Depending on how that works, it might allow us to enforce _breadcrumb only_ navigation. Until then, I think we should continue to enforce clickthrough only on mobile.

Reply:
> ios 13 may enable a top toolbar

This is on my plate too, but it's blocked by #18044 to make the change simple.

Reply:
Yep, I forgot to suggest that I think that's a totally separate issue to explore and this PR should not be blocked by it.